packages/core/EventManager.ts

```diff
@@ -908,4 +916,68 @@ export default class EventManager {
       );
     }
   }
+
+  private async createAllCRMEvents(event: CalendarEvent) {
+    const createdEvents = [];
+    for (const credential of this.crmCredentials) {
+      const crm = new CrmManager(credential);
+
+      let success = true;
+      const createdEvent = await crm.createEvent(event).catch((error) => {
+        success = false;
+      });
+      createdEvents.push({
+        type: credential.type,
+        appName: credential.appId || "",
+        uid: createdEvent?.id || "",
+        success,
+        createdEvent: {
+          id: createdEvent?.id || "",
+          uid: createdEvent?.id || "",
+          type: credential.type,
+          url: "",
+          credentialId: credential.id,
+          password: "",
+        },
+        id: createdEvent?.id || "",
+        originalEvent: event,
+        credentialId: credential.id,
+      });
+    }
```

Comment:
Not a huge fan of this but this is what the event manager is expecting

Reply:
I think we should add all props that are there and explicitly add the values as undefined/null to make it clear that certain props are intentionally left undefined and why,